scipy/optimize/lbfgsb.py

```diff
@@ -368,6 +369,7 @@ def func_and_grad(x):
     hess_inv = LbfgsInvHessProduct(s[:n_corrs], y[:n_corrs])
 
     return OptimizeResult(fun=f, jac=g, nfev=n_function_evals[0],
+                          njev=n_jac_evals[0],
```

Comment:
If it is always true that `njev==nfev` would it make sense to just use `n_function_evals[0]` again? I do see that there's precedent for having `nfev` & `njev` according to [`OptimizeResult` docs](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.OptimizeResult.html#scipy.optimize.OptimizeResult)